ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/sceneGraph/RDXSceneGraphUI.java

```diff
@@ -66,16 +69,16 @@ public RDXSceneGraphUI(ROS2PublishSubscribeAPI ros2PublishSubscribeAPI, RDX3DPan
          return uiSceneNode.getSceneNode();
       }, ros2PublishSubscribeAPI, ROS2ActorDesignation.OPERATOR);
 
-      this.panel3D = panel3D;
-
-      RDXSceneNode rootNode = new RDXSceneNode(sceneGraph.getRootNode());
-      addUISceneNode(rootNode);
+      addUISceneNode(new RDXSceneNode(sceneGraph.getRootNode()));
 
       sceneGraph.getSceneGraphSubscription().registerMessageReceivedCallback(subscriptionFrequencyText::ping);
 
       predefinedRigidBodySceneNodeBuilder = new RDXPredefinedRigidBodySceneNodeBuilder(sceneGraph);
       primitiveRigidBodySceneNodeBuilder = new RDXPrimitiveRigidBodySceneNodeBuilder(sceneGraph);
       predefinedPrimitiveRigidBodySceneNodeBuilder = new RDXPrimitiveRigidBodySceneNodeBuilder(sceneGraph);
+
+      baseUI.getPrimaryScene().addRenderableProvider(this::getRenderables);
+      baseUI.getImGuiPanelManager().addPanel(panel);
```

Comment:
These were present with every usage, so I've put them in here to clean up the usages.